File: dailyHOTEL/app/src/main/java/com/daily/dailyhotel/screen/mydaily/coupon/CouponTermInterface.java
package com.daily.dailyhotel.screen.mydaily.coupon;

import android.app.Activity;

import com.daily.dailyhotel.screen.common.web.DailyWebInterface;

public interface CouponTermInterface
{
    interface ViewInterface extends DailyWebInterface.ViewInterface
    {
    }

    interface OnEventListener extends DailyWebInterface.OnEventListener
    {
    }

    interface AnalyticsInterface extends DailyWebInterface.AnalyticsInterface
    {
        void onScreen(Activity activity, String couponIndex);
    }
}

File: dailyHOTEL/app/src/main/java/com/daily/dailyhotel/screen/mydaily/coupon/CouponTermInterface.kt
package com.daily.dailyhotel.screen.mydaily.coupon

import android.app.Activity

import com.daily.dailyhotel.screen.common.web.DailyWebInterface

interface CouponTermInterface {
    interface ViewInterface : DailyWebInterface.ViewInterface

    interface OnEventListener : DailyWebInterface.OnEventListener

    interface AnalyticsInterface : DailyWebInterface.AnalyticsInterface {
        fun onScreen(activity: Activity?, couponIndex: String?)
    }
}
